x/launch/types/request_content.go

```diff
@@ -1,4 +1,27 @@
 package types
 
+import (
+	"fmt"
+
+	codec "github.com/cosmos/cosmos-sdk/codec/types"
+)
+
 // RequestContent defines the interface for a request content
 type RequestContent interface{}
+
+// UnpackValidatorRemoval returns the ValidatorRemoval structure from the codec unpack
+func (r Request) UnpackValidatorRemoval(cdc codec.AnyUnpacker) (*ValidatorRemoval, error) {
+	if r.Content == nil {
+		return nil, fmt.Errorf("empty request content for request %d", r.RequestID)
+	}
+	var content RequestContent
+	err := cdc.UnpackAny(r.Content, &content)
+	if err != nil {
+		return nil, err
+	}
+	removeValidator, ok := content.(*ValidatorRemoval)
+	if !ok {
+		return nil, ErrFailedCodecCast
```

Comment:
Can we have an error like `not a validatorRemoval request` to be more precise?

Reply:
Also we don't need to register the error in `errors.go` if it's only used in `types` package